hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -1209,11 +1208,4 @@ public void testDatabaseLocationWithSlashInWarehouseDir() {
 
     assertThat(database.getLocationUri()).isEqualTo("s3://bucket/database.db");
   }
-
-  @Test
-  @Override
-  @Disabled("Hive currently returns an empty list instead of throwing a NoSuchNamespaceException")
-  public void testListNonExistingNamespace() {
-    super.testListNonExistingNamespace();
-  }
```

Comment:
Should we highlight this behavioral change in some doc?

Reply:
Do we have a release notes, or something? I think this is big enough behavior change to merit a line there

Reply:
there is release notes [here](https://github.com/apache/iceberg/releases), I believe it gets compiled before the release (I'm new here, so assuming).

Reply:
@nastra I see that you did the last release, is there a way I can make sure that this behavior change is included in the release?

Reply:
Marking it as resolved as @stevenzwu is aware of it now.